modules/templates/helper.go

```diff
@@ -219,6 +227,19 @@ func NewFuncMap() []template.FuncMap {
 			}
 			return dict, nil
 		},
+		"imageInfo": func(imagePath string) (*image.Config, error) {
+			resp, err := http.Get(imagePath)
```

Comment:
I don't think this is correct way to get imagine info, it should be taken from server git repo

Reply:
Agreed.